app/src/repositories/webtoon/webtoonRepository.js

```diff
@@ -1,14 +1,33 @@
 const db = require("../../config/db");
 
 class WebtoonRepository {
+  // 전체 웹툰 조회
   static async getAllWebtoons() {
-    const query = "SELECT * FROM webtoons";
+    const query = `
+    SELECT id, title, day_of_week, thumbnail_url
+    FROM webtoons
+    `;
```

Comment:
가독성을 위해 개행을 추가한 부분 좋은거같습니다.

다른분의 PR에서는 한줄로 작성된거를 봤는데 팀원들과 논의하여 컨벤션을 통일하면 좋을거같아요(저는 SQL 컨벤션에 맞게 개행 및 들여쓰기가 들어가는게 좋다고 생각하는 편입니다.)
다른분의 PR에서도 남긴 코멘트인데 쿼리 끝에 세미콜론이 추가되면 좋을거같아요https://github.com/modern-agile-team/10term-mini-back/pull/9#discussion_r2230340536
```suggestion
    SELECT id, title, day_of_week, thumbnail_url
    FROM webtoons;
    `;
```

Reply:
세미콜론 부분이 쿼리에서 없어도 잘 작동하길래 놓쳤던 것 같습니다.  말씀 주신 대로 명시적으로 표기하면 협업이나 다른 환경에서 작업할 때 더 쉽고 명확하게 사용할 수 있을 것 같습니다. 쿼리문 컨벤션 관련 부분도 팀원과 논의하여 통일된 방식으로 작성하겠습니다!